Tools/ClassIdTxtFromGbxClassInfo/Properties/launchSettings.json

```diff
@@ -2,11 +2,11 @@
   "profiles": {
     "ClassIdTxtFromGbxClassInfo": {
       "commandName": "Project",
-      "commandLineArgs": "GbxClassInfo/Vsk5OnlineLauncher.txt GbxClassInfo/Vsk5Online.txt GbxClassInfo/TrackManiaLauncher.txt GbxClassInfo/TmSunriseLauncher.txt GbxClassInfo/TmNationsESWCLauncher.txt GbxClassInfo/TmUnitedLauncher.txt GbxClassInfo/TmUnited.txt GbxClassInfo/TmForeverLauncher.txt GbxClassInfo/TmForeverServer.txt GbxClassInfo/TmForever.txt GbxClassInfo/TurboConfig.txt GbxClassInfo/Turbo.txt GbxClassInfo/ManiaplanetLauncher.txt GbxClassInfo/ManiaplanetServer.txt GbxClassInfo/ManiaplanetLogs.txt GbxClassInfo/Maniaplanet.txt"
+      "commandLineArgs": "GbxClassInfo/Vsk5OnlineLauncher.txt GbxClassInfo/Vsk5Online.txt GbxClassInfo/TrackManiaLauncher.txt GbxClassInfo/TmSunriseLauncher.txt GbxClassInfo/TmNationsESWCLauncher.txt GbxClassInfo/TmUnitedLauncher.txt GbxClassInfo/TmUnited.txt GbxClassInfo/TmForeverLauncher.txt GbxClassInfo/TmForeverServer.txt GbxClassInfo/TmForever.txt GbxClassInfo/TurboConfig.txt GbxClassInfo/Turbo.txt GbxClassInfo/ManiaplanetServer2013-05-22.txt GbxClassInfo/ManiaplanetLauncher.txt GbxClassInfo/ManiaplanetServer.txt GbxClassInfo/ManiaplanetLogs.txt GbxClassInfo/Maniaplanet.txt GbxClassInfo/TrackmaniaServer.txt"
```

Comment:
It is better to put the `ManiaplanetServer2013-05-22.txt` before all the Turbo text files, it better follows the timeline, as Turbo was released in 2016.